### PR TITLE
docs/build: document nanopi-r1 setup

### PR DESCRIPTION
**Pull request by chombourger**
Closes: #124
Signed-off-by: Cedric Hombourger <cedric.hombourger@siemens.com>

### CODE REVIEW

**Review by vj-kumar — APPROVED** [review]

**Comment by vj-kumar** [comment]
Just noticed the missing image and saw the draft. Reviewing from mobile. Will review again once ready.

File: docs/build.rst [hunk 1]
```diff
@@ -199,6 +199,129 @@ and install it::
...
+Hit ``i`` to enter the input mode and type the following configuration::
+
+    [console]
+    variant=serial
+    port=/dev/ttyGS0
+    rate=9600
+
+    [power]
+    variant=usbrelay
+    lines=959BI_1
```
Comment (hkoturap) [inline docs/build.rst:307]: @chombourger How to get the lines value.
When I connect USB relay to MTDA following information was available with dmesg

`mtda@mtda:~$ [  298.936687] usb 5-1: USB disconnect, device number 6

[  303.698841] usb 5-1: new low-speed USB device number 7 using ohci-platform

[  303.898840] usb 5-1: device descriptor read/64, error -62

[  304.241918] usb 5-1: New USB device found, idVendor=16c0, idProduct=05df, bcdDevice= 1.00

[  304.250236] usb 5-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0

[  304.257462] usb 5-1: Product: USBRelay2

[  304.261428] usb 5-1: Manufacturer: www.dcttech.com

[  304.275662] hid-generic 0003:16C0:05DF.0002: hiddev0,hidraw0: USB HID v1.01 Device [www.dcttech.com USBRelay2] on usb-1c1c400.usb-1/input`

Comment (chombourger) [reply docs/build.rst:307]: run /usr/bin/usbrelay (without any arguments), the id should be printed

Comment (hkoturap) [reply docs/build.rst:307]: Ok Got it,Thanks, Got below output and used HURTM_1
mtda@mtda:~$ sudo /usr/bin/usbrelay 

sudo: unable to resolve host mtda: No address associated with hostname
Device Found
  type: 16c0 05df
  path: /dev/hidraw0
  serial_number: 
  Manufacturer: www.dcttech.com
  Product:      USBRelay2
  Release:      100
  Interface:    0
  Number of Relays = 2
HURTM_1=1
HURTM_2=1

**Pull request merged**